projects/1.16/assets/hanami/hanami/lang/zh_cn.json

```
@@ -1 +1,77 @@
-{}
\ No newline at end of file
+{
+  "_comment": "樱木类",
+  "block.hanami.stripped_sakura_log": "条纹樱花原木",
+  "block.hanami.sakura_log": "樱花原木",
+  "block.hanami.stripped_sakura_wood": "条纹樱花木",
+  "block.hanami.sakura_wood": "樱花木",
+  "block.hanami.sakura_planks": "樱花木板",
+  "block.hanami.sakura_slab": "樱花木半砖",
+  "block.hanami.sakura_stairs": "樱花木楼梯",
+  "block.hanami.sakura_pressure_plate": "樱花木压力板",
+  "block.hanami.sakura_fence": "樱花木栅栏",
+  "block.hanami.sakura_fence_gate": "樱花木栅栏门",
+  "block.hanami.sakura_button": "樱花木按钮",
+  "block.hanami.sakura_trapdoor": "樱花木活板门",
+  "block.hanami.sakura_door": "樱花木门",
+  "block.hanami.sakura_sign": "樱花木告示牌",
+  "block.hanami.sakura_leaves": "樱花树叶",
+  "block.hanami.sakura_sapling": "樱花树苗",
+  "block.hanami.sakura_leaf_carpet": "樱花叶地毯",
+  "block.hanami.sakura_bookshelf": "樱花木书架",
+  "block.hanami.sakura_chest": "樱花木箱子",
+  "block.hanami.sakura_trapped_chest": "樱花木陷阱箱",
+  "block.hanami.sakura_ladder": "樱花木梯子",
+  "block.hanami.sakura_vertical_slab": "樱花木竖半砖",
+  "block.hanami.vertical_sakura_planks": "竖樱花木板",
```

```suggestion
  "block.hanami.sakura_vertical_slab": "樱花木竖直台阶",
  "block.hanami.vertical_sakura_planks": "竖纹樱花木板",
```
同步quark的已有译名
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/14b1cc24dd37e10275266058a94c5110d3fbc55e/projects/1.16/assets/quark/quark/lang/zh_cn.json#L626-L627